mammoth/train_single.py

```diff
@@ -83,7 +83,8 @@ def main(
     batch_queue=None,
     semaphore=None,
     task_queue_manager=None,
-    checkpoint=None,
+    frame_checkpoint=None,
+    frame_ckpt_path=None,
```

Comment:
let's have a consistent naming scheme, pick either
 `frame_ckpt_path` -> `frame_checkpoint_path`
 or 
`frame_checkpoint` -> `frame_ckpt`

Reply:
The `ckpt_path` naming is a remnant from the old saving code. I'll run a replace all.